mvp/README.md

```diff
@@ -0,0 +1,19 @@
+
+**User-Centric** design enables the continuous delivery of value to users. This means that putting a good, functional, tool in the hands of users quickly is more valuable than delivering a perfect tool when it's too late.
```

Comment:
getting too late in delivering the product may reduce the interest of users in the tool. So delivering at right time is important.